### PR TITLE
FEATURE: ONE-2732 Custom label rotation

### PR DESCRIPTION
**Pull request by ZdenekVidensky**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/253/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1495/ : SUCCESS in 4m 09s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/255/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1498/ : FAILURE in 51s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/256/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1499/ : SUCCESS in 2m 20s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/257/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1500/ : SUCCESS in 1m 48s

**Comment by ivan-nejezchleb** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/260/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 9s
- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED

**Comment by ZdenekVidensky** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/261/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 9s
- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/275/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1514/ : SUCCESS in 4m 03s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/278/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1519/ : SUCCESS in 3m 48s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/280/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1521/ : SUCCESS in 6m 20s

File: src/components/visualizations/chart/highcharts/customConfiguration.ts [hunk 1]
```diff
@@ -620,12 +614,12 @@ function getAxesConfiguration(chartOptions: any) {
             }
 
             let visible: boolean;
+            let rotation: string;
```
Comment (peterlacko) [inline src/components/visualizations/chart/highcharts/customConfiguration.ts]: use const here, I don't see re-assigning of these variables

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/281/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1522/ : SUCCESS in 2m 02s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/50/ : SUCCESS in 16s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1524/ : FAILURE in 8m 07s
- https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/627/ : UNSTABLE in 25m 23s

**Comment by peterlacko** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/628/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/287/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/516/ : UNSTABLE in 15m 55s

**Comment by ZdenekVidensky** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/288/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1528/ : FAILURE in 5m 22s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/629/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/289/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/517/ : SUCCESS in 23m 06s

**Comment by ZdenekVidensky** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/292/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1530/ : FAILURE in 3m 36s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/630/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/293/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/518/ : UNSTABLE in 22m 32s

**Comment by ZdenekVidensky** [comment]
extended test storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/294/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1531/ : SUCCESS in 1m 58s

**Comment by yenkins** [comment]
Demo is running at https://client-demo.na.intgdc.com:50016/storybook/

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/295/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/519/ : FAILURE in 7m 11s

**Comment by ZdenekVidensky** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/631/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/296/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/520/ : SUCCESS in 15m 29s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/297/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1532/ : SUCCESS in 1m 55s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/298/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1533/ : SUCCESS in 4m 14s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/51/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1535/ : SUCCESS in 3m 52s
- https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/632/ : SUCCESS in 16m 58s

**Pull request merged**